frontend/app/routes/public/renew/$id/adult-child/confirm-email.tsx

```diff
@@ -170,7 +170,6 @@ export default function RenewAdultChildConfirmEmail() {
         <fetcher.Form method="post" noValidate>
           <input type="hidden" name="_csrf" value={csrfToken} />
           <fieldset className="mb-6">
-            <legend className="mb-4 font-lato text-2xl font-bold">{t('renew-adult-child:confirm-email.email-header')}</legend>
```

Comment:
I believe we can remove this parent `<fieldset>` now as there's no corresponding `<legend>` for it.

(same for `/confirm-phone`).

Reply:
true.  We still need margin and it doesn't appear that the Input component accepts additional styles as props, so I'll just keep the div in place of fieldset.